data/yaml/name_variants.yaml

```diff
@@ -1403,6 +1403,11 @@
 - canonical: {first: Jose, last: Camacho-Collados}
   variants:
   - {first: José, last: Camacho-Collados}
+- canonical: {first: Aaron Steven, last: White}
+  orcid: 0000-0003-0057-9246
+  institution: University of Maryland
+  variants:
+  - {first: Aaron, last: White}
```

Comment:
Should we even add the name variant? `rg "<first>Aaron</first><last>White"` returns zero results on this branch.